app/packages/looker/src/lookers/video.ts

```diff
@@ -564,19 +396,25 @@ export class VideoLooker extends AbstractLooker<VideoState, VideoSample> {
   }
 
   private hasFrame(frameNumber: number) {
+    if (frameNumber === this.firstFrameNumber) {
+      return this.firstFrame;
```

Comment:
this is also present in `this.frames` right? any reason to shortcut here?

Reply:
I don't believe the first frame is added to the LRU cache. That's something I removed because we never want to delete it